Heal fedora's copr build

Current master fails copr build: https://travis-ci.org/pgjdbc/pgjdbc/jobs/142483147
Not sure how `maven-enforcer-plugin` got there previously

```
[INFO] Reactor Summary:
[INFO] 
[INFO] PostgreSQL JDBC Driver - JDBC 4.2 .................. FAILURE [  0.002 s]
[INFO] PostgreSQL JDBC Driver aggregate ................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.435 s
[INFO] Finished at: 2016-07-05T15:33:09+00:00
[INFO] Final Memory: 12M/171M
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin org.apache.maven.plugins:maven-enforcer-plugin:1.3.1 or one of its dependencies could not be resolved: Cannot access central (https://repo.maven.apache.org/maven2) in offline mode and the artifact org.apache.maven.plugins:maven-enforcer-plugin:jar:1.3.1 has not been downloaded from it before. -> [Help 1]
```

/cc @praiskup

--- a/packaging/rpm/postgresql-jdbc.spec.tpl
+++ b/packaging/rpm/postgresql-jdbc.spec.tpl
@@ -66,6 +66,7 @@ BuildRequires:	jpackage-utils
 BuildRequires:	maven-local
 BuildRequires:	java-comment-preprocessor
 BuildRequires:	properties-maven-plugin
+BuildRequires:	maven-enforcer-plugin
 BuildRequires:	maven-plugin-bundle
 BuildRequires:	maven-plugin-build-helper
 

